layouts/shortcodes/integrations.html

```diff
@@ -69,70 +79,39 @@
 
 <!-- using 5 col -->
 <div class="row no-gutters integration-tiles mt-4" data-ref="container">
-    {{ range $i, $v := $integration_list }}
-
-        {{ $.Scratch.Set "formatname" ""}}
-        {{ if $v.Params.integration_title }}
-            {{ $.Scratch.Set "formatname" $v.Params.integration_title }}
-        {{ else if $v.Params.name }}
-            {{ $.Scratch.Set "formatSlice" (slice)}}
-            {{ range $i, $word := (split $v.Params.name "_") }}
-                {{ $.Scratch.Add "formatSlice" (print (substr $word 0 1 | upper) (substr $word 1 (len $word))) }}
-            {{ end }}
-            {{ $.Scratch.Set "formatname" (delimit ($.Scratch.Get "formatSlice") " ") }}
-        {{ end }}
-        {{ $formatname := $.Scratch.Get "formatname" }}
-
-        {{ $urlname := $v.File.TranslationBaseName }}
-        {{ $src := (print "images/integrations_logos/" ($urlname | lower) ".png")}}
-        {{ $indx := $src }}
-
-        {{/* if  $src */}}
-        {{ if and $indx (fileExists (print "static/" $src)) }}
-            {{ if $formatname }}
-            <div id="mixid_{{$i}}" class="col-6 col-sm-4 col-md-3 col-lg-3 col-xl-2-4 mix text-center {{ range $i, $e := $v.Params.categories }} cat-{{ replace $e "/" "" | urlize }} {{ end }}" data-id="{{ $i }}" data-ref="item">
-                <div id="{{ $v.Params.name | default ($urlname | lower) }}" class="card">
-                    {{ if in $v.Params.categories "marketplace" }}
+    {{ range $integration := $integrationData }}
+        {{ if $integration.int_logo }}
+            <div id="mixid_{{ $integration.id }}" class="col-6 col-sm-4 col-md-3 col-lg-3 col-xl-2-4 mix text-center {{ range $cat := $integration.tags }} {{ $cat }} {{ end }}" data-id="{{ $integration.id }}" data-ref="item">
```

Comment:
do we need to worry about sanitizing $cat here? previous output had a `urlize` filter

Reply:
So this is actually already happening when we create the `$integration.tags` data structure further up the file. In general i tried to pre format the data in the creation step over when we output. so we can just plainly output the variables mostly.